README.md

```diff
@@ -72,6 +72,8 @@ type NFT = {
 - `rentalStatus`: Filter results by `Rental status`. Can be used only when querying NFTs that are on rent using the `isOnRent` flag. Possible values: `open`, `executed`, `claimed`, `cancelled`.
 - `minPrice`: Filter results by minimun price. Type: `number`
 - `maxPrice`: Filter results by max price. Type: `number`
+- `minEstateSize`: Filter results by minium `searchEstateSize`. Type: `number`
```

Comment:
The `searchEstateSize` seems to me too implementation-specific, for someone that does not know the subgraph schema that might not mean much. What do you think as simply saying something like "Filter results by the minimum amount of parcels in the Estate". Same for the `maxEstateSize` below.